js_modules/dagster-ui/packages/ui-components/src/components/Button.tsx

```diff
@@ -160,7 +159,7 @@ export const intentToTextAndIconColor = (intent: BlueprintIntent) => {
   if (intent === 'primary') {
     return Colors.accentReversed();
   }
-  return CoreColors.White;
+  return Colors.alwaysWhite();
```

Comment:
I like this naming, was actually wondering just the other day whether white and black were static. 🙌